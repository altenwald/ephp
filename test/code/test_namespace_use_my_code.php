<?php

include_once(__DIR__ . "/my_code.php");

use const MyCode\WHO;
use function MyCode\singleton;

use MyCode as MC;

singleton();
$hello = new MC\Hello;

MC\singleton();

