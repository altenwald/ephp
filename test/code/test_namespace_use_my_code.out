MyCode\Hello::__construct -- Hello world!
MyCode\Hello::__construct -- Hello world!
Hello already created!
